rosie/src/main/java/com/karumi/rosie/domain/usecase/UseCaseHandler.java

```diff
@@ -29,6 +33,10 @@
   private final TaskScheduler taskScheduler;
   private final ErrorHandler errorHandler;
 
+  // This list is used to retain error adapters in a non-weak reference associated with this use
+  // case handler
+  private final List<OnErrorCallback> errorCallbacks = new ArrayList<>();
```

Comment:
I can, but I don't know if I'll  have time soon. Do you want to merge this and release a new version with the fix soon?